api/proto/info/info.proto

```diff
@@ -682,6 +682,8 @@ enum AttestationState {
 enum AppInstMetaDataType {
   APP_INST_META_DATA_TYPE_NONE = 0;
   APP_INST_META_DATA_TYPE_KUBE_CONFIG = 1;
+  APP_INST_META_DATA_TYPE_CURRENT_VERSION = 2;
+  APP_INST_META_DATA_CUSTOM_STATUS = 3;
```

Comment:
Are you expecting applications to use both the new version and status? I thought status would be used by the applications to include version as well as status.

Reply:
as I mentioned in the commit message:

```
The patch also adds APP_INST_META_DATA_TYPE_CURRENT_VERSION, which
technically doesn't belong to this modification, but it was forgoten
when we worked on kubeconfig. This patch adds it in order to
APP_INST_META_DATA_CUSTOM_STATUS work properly
```

Reply:
ok, seems like my residual memory has failed me. I will remove APP_INST_META_DATA_TYPE_CURRENT_VERSION